src/ctap2.rs

```diff
@@ -1185,12 +1185,7 @@ impl<UP: UserPresence, T: TrussedRequirements> crate::Authenticator<UP, T> {
             self.state
                 .runtime
                 .rotate_key_agreement_key(&mut self.trussed);
-            if self.state.persistent.retries() == 0 {
-                return Err(Error::PinBlocked);
-            }
-            if self.state.persistent.pin_blocked() {
-                return Err(Error::PinAuthBlocked);
-            }
+            self.state.pin_blocked()?;
```

Comment:
Having 3 methods in `state.rs` called `pin_blocked` is confusing. Makes me wish for jump to definition in code review

Reply:
was confused by the very same and had to look it up manually - but still makes sense as it is I guess